AutomatedTesting/Gem/PythonTests/Atom/atom_utils/atom_tools_utils.py

```diff
@@ -11,8 +11,14 @@
 import azlmbr.atom
 import azlmbr.atomtools
 import azlmbr.bus as bus
+import azlmbr.paths
 
+MATERIAL_TYPES_PATH = os.path.join(
```

Comment:
nit- It's not hurting anything but I'm not sure these path constants belong here since they are not generally available to all the tools.

Reply:
its not an issue for now and even if the paths aren't present on the machine running the test then as long as the test isn't utilizing any of the path constants it shouldn't hurt anything
if this starts to grow more robust then we can explore moving it into something like a class but i think its fine for now since there aren't many paths needed by the tests yet